frontend/src/pages/WorkspaceTeam/WorkspaceTeam.tsx

```diff
@@ -1,26 +1,62 @@
+import Tabs from '@components/Tabs/Tabs'
 import { useGetWorkspaceAdminsQuery } from '@graph/hooks'
 import { AdminRole, WorkspaceAdminRole } from '@graph/schemas'
-import { Box } from '@highlight-run/ui'
+import { Badge, Box, IconSolidUserAdd, Stack, Text } from '@highlight-run/ui'
 import AllMembers from '@pages/WorkspaceTeam/components/AllMembers'
 import AutoJoinForm from '@pages/WorkspaceTeam/components/AutoJoinForm'
 import InviteMemberModal from '@pages/WorkspaceTeam/components/InviteMemberModal'
 import { Authorization } from '@util/authorization/authorization'
 import { useParams } from '@util/react-router/useParams'
 import React from 'react'
+import { useNavigate } from 'react-router-dom'
 import { useToggle } from 'react-use'
 
 import Button from '../../components/Button/Button/Button'
 import layoutStyles from '../../components/layout/LeadAlignLayout.module.scss'
 import styles from './WorkspaceTeam.module.scss'
 
+type MemberKeyType = 'members' | 'invites'
+
 const WorkspaceTeam = () => {
-	const { workspace_id } = useParams<{ workspace_id: string }>()
+	const { workspace_id, member_tab_key = 'members' } = useParams<{
+		workspace_id: string
+		member_tab_key?: MemberKeyType
+	}>()
 	const { data, error, loading } = useGetWorkspaceAdminsQuery({
 		variables: { workspace_id: workspace_id! },
 		skip: !workspace_id,
 	})
+	const navigate = useNavigate()
 	const [showModal, toggleShowModal] = useToggle(false)
 
+	const TabContentContainer = ({ children }: { children: any }) => (
+		<Box mt="8">
+			<div className={styles.tabTitleContainer}>
+				<h4 className={styles.tabTitle}>
+					{determineTitle(member_tab_key)}
+				</h4>
+				<InviteMemberModal
+					workspaceId={workspace_id}
+					workspaceName={data?.workspace?.name}
+					workspaceInviteLinks={data?.workspace_invite_links}
+					showModal={showModal}
+					toggleShowModal={toggleShowModal}
+				/>
+				<Button
+					className={styles.inviteButton}
+					trackingId="WorkspaceTeamInviteMember"
+					type="primary"
+					onClick={toggleShowModal}
+					small
+				>
+					<IconSolidUserAdd />
+					<Text>Invite users</Text>
+				</Button>
```

Comment:
If you import `Button` from `components/Button` instead of `components/Button/Button/Button` you should be able to use the `iconLeft` prop to display the icon and text the way you want it here.

That button is also up to date with the latest mocks in Figma so the props will match up better too (e.g. `size="small"`, `kind="primary"`, and `emphasis="high"`).

<img width="503" alt="image" src="https://github.com/highlight/highlight/assets/308182/8dd0ba3f-5376-4063-93e3-fe97264ff849">

Reply:
Good to know! Thanks for sharing